app/src/main/res/layout/dialog_date_range_home.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+    <androidx.coordinatorlayout.widget.CoordinatorLayout
```

Comment:
@danacealfonso 
Layout for the dialog background is unnecessary.

Reply:
Change in e69d716cd680664f4de24a4ecd003d9136f45f75